dist/index.html

```diff
@@ -14,5 +14,7 @@ <h1>To Do List</h1>
     <div class="container">
         <!--This is where items will go-->
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you add your own logo (Not Microverse logo) at the top of your readme file to make yourself a brand in the tech industry to catch the eyes of recruiters 👍